ARP_hwSrc: expected '0:11:22:33:44:55', got '00:11:22:33:44:55'
ARP_hwDst: expected 'aa:bb:cc:dd:ee:ff', got 'AA:BB:CC:DD:EE:FF'
FAIL runtime_unit_impcap_arp_parser (exit status: 1)

--- a/tests/unit/impcap_arp_parser_test.c
+++ b/tests/unit/impcap_arp_parser_test.c
@@ -209,8 +209,8 @@ static void test_valid_arp_still_parses_
     CHECK(ret->size == 0);
     free(ret);
 
-    check_json_string(j, "ARP_hwSrc", "0:11:22:33:44:55");
-    check_json_string(j, "ARP_hwDst", "aa:bb:cc:dd:ee:ff");
+    check_json_string(j, "ARP_hwSrc", "00:11:22:33:44:55");
+    check_json_string(j, "ARP_hwDst", "AA:BB:CC:DD:EE:FF");
     check_json_string(j, "ARP_pSrc", "192.0.2.1");
     check_json_string(j, "ARP_pDst", "198.51.100.2");
 
@@ -228,8 +228,8 @@ static void test_valid_rarp_still_parses
     CHECK(ret->size == 0);
     free(ret);
 
-    check_json_string(j, "RARP_hwSrc", "0:11:22:33:44:55");
-    check_json_string(j, "RARP_hwDst", "aa:bb:cc:dd:ee:ff");
+    check_json_string(j, "RARP_hwSrc", "00:11:22:33:44:55");
+    check_json_string(j, "RARP_hwDst", "AA:BB:CC:DD:EE:FF");
     check_json_string(j, "RARP_pSrc", "192.0.2.1");
     check_json_string(j, "RARP_pDst", "198.51.100.2");
 
